packages/blockchain-wallet-v4-frontend/src/data/components/interest/sagas.ts

```diff
@@ -377,6 +377,20 @@ export default ({
       yield put(
         actions.analytics.logEvent(INTEREST_EVENTS.DEPOSIT.SEND_SUCCESS)
       )
+
+      const afterTransactionR = yield select(
+        selectors.components.interest.getAfterTransaction
+      )
+      const afterTransaction = afterTransactionR.getOrElse({
+        show: false
+      } as InterestAfterTransactionType)
+      if (afterTransaction && afterTransaction.show) {
```

Comment:
idk if you want to check both, but you could do: 
```suggestion
      if (afterTransaction?.show) {
```

Reply:
yeah forgot to use this way, will update ;)

Reply:
何をしたいの私からお金を巻き上げ落としてるわけたりみんな 英語分からないんだけど私